Meter.c

```diff
@@ -158,19 +180,36 @@ static void BarMeterMode_draw(Meter* this, int x, int y, int w) {
          value = MINIMUM(value, this->total);
          blockSizes[i] = ceil((value / this->total) * w);
          blockSizes[i] = MINIMUM(blockSizes[i], w - offset);
+
+#ifdef HAVE_LIBNCURSESW
+         extraWidth = (int)ceil((value / this->total) * w * barLen) % barLen;
+#endif
```

Comment:
I think this needs some minor update. The `extraWidth` calculation probably should re-use the value calculated in the `blockSizes[i]` assignments above.